app/views/vm_common/_reconfigure.html.haml

```diff
@@ -206,10 +215,33 @@
               {{disk.hdMode}}
             %td{"ng-if" => @reconfigitems.first.vendor == 'vmware'}
               {{disk.new_controller_type}}
-            %td
+            %td{"ng-if" => "disk.add_remove != 'resizing'"}
               {{disk.hdSize}}
-            %td
+            %td{"ng-if" => "disk.add_remove == 'resizing'"}
+              %input.form-control{"type"        => "text",
+                                  "id"          => "dvcResize",
+                                  "name"        => "dvcResize",
+                                  "ng-model"    => "disk.hdSize",
+                                  "required"    => "",
+                                  "ng-pattern"  => "hdpattern",
+                                  "placeholder" => "Enter Size",
+                                  "ng-change"   => "vm.hdChanged()",
+                                  "validate-multiple" => "1",
+                                  :miqmin             => "{{ (disk.orgHdUnit=='GB')  && (1024*disk.orgHdSize+1) || (1*disk.orgHdSize+1)}}",
```

Comment:
CC won't warn here, but please keep the same rules...

`:miqmin             => "{{ (disk.orgHdUnit === 'GB')  && (1024 * disk.orgHdSize + 1) || (1 * disk.orgHdSize + 1) }}",`

Reply:
@himdel Sorry, over looked this one. Should be fix now.